packages/editor-ui/src/stores/credentials.ts

```diff
@@ -1,3 +1,4 @@
+import { INodeUi } from './../Interface';
```

Comment:
```suggestion
import type { INodeUi } from './../Interface';
```
Also non-relative import?